src/pages/signup-affiliates/components/_account-address.tsx

```diff
@@ -90,20 +90,30 @@ const AccountAddress = ({
     ]
 
     useEffect(() => {
-        updateData({ ...form_data })
+        let data = {} as AffiliateAccountTypes['account_address']
+        for (const property in form_data) {
+            data = {
+                ...data,
+                [property]:
+                    typeof form_data[property] === 'string'
+                        ? form_data[property].trim()
```

Comment:
If user can write special character which not supported on API, let's wrap field with `customSlugify` in `Submit` function